src/db/index.js

```diff
@@ -1,17 +1,17 @@
-const Sequelize = require('sequelize');
-const users = require('./models/User');
-const fs = require('fs');
-const path = require('path');
+const Sequelize = require("sequelize");
```

Comment:
Let's make sure we get to changing these soon

Reply:
Done, commit 8629868.. missed this dir